.github/workflows/deploy-env.yml

```diff
@@ -11,17 +11,43 @@ on:
         description: "The Elastic Beanstalk version label"
         required: true
         type: string
+      notify_honeybadger:
+        description: "Whether or not to notify HoneyBadger"
+        required: false
+        type: boolean
+      notify_new_relic:
+        description: "Whether or not to notify New Relic"
+        required: false
+        type: boolean
     secrets:
       EB_AWS_ACCESS_KEY_ID:
+        description: "The AWS Access Key ID you would like to deploy with (e.g. prod or staging)"
         required: true
       EB_AWS_SECRET_ACCESS_KEY:
+        description: "The AWS Secret Access Key you would like to deploy with (e.g. prod or staging)"
         required: true
       DEPLOY_FAILURES_SLACK_WEBHOOK_URL:
+        description: "The channel to message in on failures"
```

Comment:
Maybe something like:

The Slack webhook url that maps to a Slack channel. This channel will be notified after a successful deploy.

Might also be worth mentioning that it should be secret.
i.e. https://www.darkreading.com/cloud/slack-s-incoming-webhooks-can-be-weaponized-in-phishing-attacks

Reply:
Thanks! I'll update